inst/tutorials/r_and_rstudio_basic/r_and_rstudio_basic.Rmd

```diff
@@ -77,10 +88,17 @@ You can open this project in the future in one of three ways:
 - Switch among projects by clicking on the R project symbol in the upper left
 corner of RStudio
 
-
+```{r quiz: R Projects, echo=FALSE}
+question("What is not a benefit of using RStudio projects?",
+    answer("All of your files and outputs are linked to the project directory"),
+    answer("R automatically looks for files in the project directory so you don't have to specify a full file path"),
+    answer("When you reopen a project, your code is saved so all you need to do is rerun it", correct=TRUE)
+)
```

Comment:
I'm not sure how useful it is having quizzes for this kind of content. What do you think, Cathy and Stephan? Though I can appreciate how it might be useful in getting someone to learn about RStudio projects, I might just give a few use cases, link to the docs, and leave it at that.